FlowCrypt/src/main/java/com/flowcrypt/email/ui/activity/fragment/base/BaseFragment.kt

```diff
@@ -37,102 +37,103 @@ import com.google.android.material.snackbar.Snackbar
 
+  override fun onCreate(savedInstanceState: Bundle?) {
+    super.onCreate(savedInstanceState)
+    LogsUtil.d(loggingTag, "onCreate")
+  }
+
   override fun onCreateView(
     inflater: LayoutInflater,
     container: ViewGroup?,
     savedInstanceState: Bundle?
   ): View? {
-    return if (contentResourceId > 0) {
-      inflater.inflate(contentResourceId, container, false)
-    } else super.onCreateView(inflater, container, savedInstanceState)
+    LogsUtil.d(loggingTag, "onCreateView")
+    binding = inflateBinding(inflater, container)
+    return binding?.root
   }
 
   override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
     super.onViewCreated(view, savedInstanceState)
+    LogsUtil.d(loggingTag, "onViewCreated")
+    doBaseUISetup(this)
     initAccountViewModel()
   }
 
-  override fun onDetach() {
-    super.onDetach()
-    lifecycle.removeObserver(connectionLifecycleObserver)
+  override fun onStart() {
+    super.onStart()
+    LogsUtil.d(loggingTag, "onStart")
   }
 
-  open fun onAccountInfoRefreshed(accountEntity: AccountEntity?) {
+  override fun onResume() {
+    super.onResume()
+    LogsUtil.d(loggingTag, "onResume")
+  }
 
+  override fun onCreateOptionsMenu(menu: Menu, inflater: MenuInflater) {
+    super.onCreateOptionsMenu(menu, inflater)
+    LogsUtil.d(loggingTag, "onCreateOptionsMenu")
   }
 
-  /**
-   * This method handles a success result of some loader.
-   *
-   * @param loaderId The loader id.
-   * @param result   The object which contains information about the loader results
-   */
-  open fun onSuccess(loaderId: Int, result: Any?) {
+  override fun onPrepareOptionsMenu(menu: Menu) {
+    super.onPrepareOptionsMenu(menu)
+    LogsUtil.d(loggingTag, "onPrepareOptionsMenu")
+  }
 
+  override fun onPause() {
+    super.onPause()
+    LogsUtil.d(loggingTag, "onPause")
   }
 
-  /**
-   * This method handles a failure result of some loader. This method contains a base
-   * realization of the failure behavior.
-   *
-   * @param loaderId The loader id.
-   * @param e        The exception which happened when loader does it work.
-   */
-  open fun onError(loaderId: Int, e: Exception?) {
-    e?.message?.let {
-      if (view != null) {
-        UIUtil.showInfoSnackbar(requireView(), it)
-      }
-    }
+  override fun onSaveInstanceState(outState: Bundle) {
+    super.onSaveInstanceState(outState)
+    LogsUtil.d(loggingTag, "onSaveInstanceState")
+  }
+
+  override fun onStop() {
+    super.onStop()
+    LogsUtil.d(loggingTag, "onStop")
+  }
+
+  override fun onDestroyView() {
+    super.onDestroyView()
+    LogsUtil.d(loggingTag, "onDestroyView")
+    binding = null
+  }
+
+  override fun onDestroy() {
+    super.onDestroy()
+    LogsUtil.d(loggingTag, "onDestroy")
+  }
+
+  override fun onDetach() {
+    super.onDetach()
+    LogsUtil.d(loggingTag, "onDetach")
+    lifecycle.removeObserver(connectionLifecycleObserver)
+  }
+
+  open fun onAccountInfoRefreshed(accountEntity: AccountEntity?) {
+
```

Comment:
is it intentionally empty? If so, please put a comment like `// nothing to do here`